thali/thalicryptomanager.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var crypto = require('crypto');
+var fs = require('fs');
+var path = require('path');
+
+// The password is not secure because anyone who can get to the file can get
+// to the app and thus can get the password. The password is used here only to
+// satisfy the crypto/PKCS12 APIs.
+var password = 'password';
+var certname = 'certname';
+var country = 'country';
+var organization = 'organization';
+var pkcs12FileName = '/pkcs12.pfx';
+var macName = 'SHA256';
+var hashSizeInBytes = 16;
+
+module.exports = {
+  
+  pkcs12FileName: pkcs12FileName,
```

Comment:
These are local variables in cryptomanager.js that are needed when creating the pkcs12 package.

      var pkcs12Content = crypto.pkcs12
        .createBundle(password, certname, country, organization);